packages/web-components/src/components/cbp-structured-list/cbp-structured-list.scss

```diff
@@ -109,4 +109,35 @@ cbp-structured-list {
   div[role=list]:only-child > *:first-child {
     border-top: var(--cbp-border-size-md) solid var(--cbp-structured-list-border-color);
   }
+
+  /** Selectable */
+  cbp-structured-list-item cbp-checkbox{
+    display: inline-flex;
```

Comment:
This shouldn't be needed. If I uncheck it in the browser tools, I see no change.